### PR TITLE
Refactor files to adhere to AppDev template

### PR DESCRIPTION
**Pull request by manishsaha**
## Overview
Added a basic Hello World controller and refactored codebase to adhere to AppDev's template. Also added basic dao logic along with existing code.

## Changes Made
As above.

## Test Coverage
Ran locally and made sure I was able to hit the hello world endpoint.

## Next Steps 
Work on sketching endpoints for coursegrab.

### CODE REVIEW

File: src/app/coursegrab/dao/courses_dao.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+def get_course_by_catalog_num(catalog_num):
+    return Course.query.filter(Course.catalog_num == catalog_num).first()
+
+
+def create_courses(course_lst):
+    for (subject_code, course_num, title, catalog_num, section) in course_lst:
+        course = Course(
+            subject_code=subject_code, course_num=course_num, title=title, catalog_num=catalog_num, section=section
```
Comment (kq-li) [inline src/app/coursegrab/dao/courses_dao.py]: wtf black does this?

File: src/run.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from app import app
+from os import environ
+
+if __name__ == "__main__":
+    HOST = "0.0.0.0"
+    PORT = environ["PORT"]
+    print("Server running on {}:{}".format(HOST, PORT))
+    app.run(host="localhost", port=PORT)
```
Comment (kq-li) [inline src/run.py]: change the host here

File: src/run.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from app import app
+from os import environ
+
+if __name__ == "__main__":
+    HOST = "0.0.0.0"
```
Comment (kq-li) [inline src/run.py]: prefer lowercase here, uppercase is reserved for constants

File: src/app/coursegrab/dao/courses_dao.py (hunk 1 above)
Comment (manishsaha) [reply src/app/coursegrab/dao/courses_dao.py]: yea lmao

File: src/run.py (hunk 3 above)
Comment (manishsaha) [reply src/run.py]: isn't host tech a constant

Comment (kq-li) [reply src/run.py]: global constants*

Comment (kq-li) [reply src/run.py]: i also don't super feel like this print message is necessary so if you remove it you can just inline these vars but either way is fine

**Pull request merged**